packages/core/src/tracing/transaction.ts

```diff
@@ -13,6 +11,9 @@ import type {
 } from '@sentry/types';
 import { dropUndefinedKeys, logger } from '@sentry/utils';
 
+import { DEFAULT_ENVIRONMENT } from '../constants';
+import type { Hub } from '../hub';
```

Comment:
l: Do we want to point these to the actual definitions instead of to index?

Reply:
Changed the imports to use `../hub` - does this still apply?

Reply:
nope all good